mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -92,8 +92,14 @@ static void ProcessExit (object sender, EventArgs e)
 
 		static GDIPlus ()
 		{
+#if NETSTANDARD1_6
+			bool isUnix = RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```

Comment:
I just thought about this, I think `bool isUnix = !RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` is better since it'll work on most platforms that'll show up in the future, like FreeBSD etc.